canonical-web-and-design/ubuntu.com
# update col/row markup on engage pages

## Done

- Updated all engage pages in line with Vanilla 2's new grid pattern (see point 2 of [the upgrade guide](https://ubuntu.com/blog/vanilla-framework-2-0-upgrade-guide))

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Look at each engage page and ensure grid/column layout looks correct (blockquotes, stepped lists and navigation aren't addressed in this PR):
  - [x] /engage
  - [x] /engage/14-04-esm.html
  - [x] /engage/19-04-webinar.html
  - [x] /engage/451-automotive.html
  - [x] /engage/451-cloud-economics.html
  - [x] /engage/ITstrategen-case-study.html
  - [x] /engage/_heading-help.html
  - [x] /engage/ai-gettingstarted.html
  - [x] /engage/ai-ml-ubuntu.html
  - [x] /engage/apparmor-intro.html
  - [x] /engage/base_engage.html
  - [x] /engage/casestudy-azeti.html
  - [x] /engage/casestudy-azlogica.html
  - [x] /engage/casestudy-botsandus.html
  - [x] /engage/casestudy-cinergy.html
  - [x] /engage/casestudy-city-network.html
  - [x] /engage/casestudy-commercetools.html
  - [x] /engage/casestudy-imsevolve.html
  - [x] /engage/casestudy-innovapos.html
  - [x] /engage/casestudy-nexiona.html
  - [x] /engage/casestudy-rocketchat.html
  - [x] /engage/casestudy-sensape.html
  - [x] /engage/cloud-economics.html
  - [x] /engage/cloud-fray.html
  - [x] /engage/cloud-init-whitepaper.html
  - [x] /engage/core18-security.html
  - [x] /engage/cto-guide-sdn-nfv-vnf.html
  - [x] /engage/data-pipelines-kubeflow.html
  - [x] /engage/deploy-ai-ml-model.html
  - [x] /engage/developer-desktop.html
  - [x] /engage/devops-cicd-webinar.html
  - [x] /engage/devops-iot-webinar.html
  - [x] /engage/ebook-maas.html
  - [x] /engage/edge-month.html
  - [x] /engage/essential-guide-big-data-solutions.html
  - [x] /engage/events.html
  - [x] /engage/evolving-software.html
  - [x] /engage/financial-services-month.html
  - [x] /engage/fingbox-case-study.html
  - [x] /engage/finserv-multi-cloud.html
  - [x] /engage/future-proof-iot.html
  - [x] /engage/hiri-case-study.html
  - [x] /engage/index.html
  - [x] /engage/industrial-drones-case-study.html
  - [x] /engage/industrial-drones.html
  - [x] /engage/iot-business-model.html
  - [x] /engage/iot-devops.html
  - [x] /engage/iot-disk-encryption.html
  - [x] /engage/k8s-allan-gray.html
  - [x] /engage/kernel-telco-nfv.html
  - [x] /engage/low-latency-report.html
  - [x] /engage/lxd-whitepaper.html
  - [x] /engage/maas-hardware-testing.html
  - [x] /engage/microsoft-active-directory.html
  - [x] /engage/modern-cloud.html
  - [x] /engage/multi-cloud-guide.html
  - [x] /engage/nfv-sdn-openstack.html
  - [x] /engage/northstar.html
  - [x] /engage/openstack-made-easy.html
  - [x] /engage/openstack-security-webinar.html
  - [x] /engage/openstack-security-whitepaper.html
  - [x] /engage/openstack-telco-clouds.html
  - [x] /engage/private-cloud.html
  - [x] /engage/public-cloud.html
  - [x] /engage/robot-operating-system-choice.html
  - [x] /engage/shift-to-app-store-experience.html
  - [x] /engage/small-robot-company.html
  - [x] /engage/snap-deltas.html
  - [x] /engage/starting-with-ai.html
  - [x] /engage/stuckstack-ebook.html
  - [x] /engage/telco-cloudification-ebook.html
  - [x] /engage/ubuntu-18-10.html
  - [x] /engage/ubuntu-compliance.html
  - [x] /engage/ubuntu-core-and-kura.html
  - [x] /engage/ubuntu-lime-telco.html
  - [x] /engage/vmware-to-openstack.html
  - [x] /engage/whats-new-ubuntu-14-04-lts.html
  - [x] /engage/whitepaper-containers.html
  - [x] /engage/whitepaper-iot-security.html
  - [x] /engage/de/openstack-made-easy.html
  - [x] /engage/de/openstack-security-de.html
  - [x] /engage/de/vmware-to-openstack.html
  - [x] /engage/de/what-can-you-do-with-kubernetes.html
  - [x] /engage/fr/esm.html                 
  - [x] /engage/fr/esm_1404.html            
  - [x] /engage/fr/openstack-made-easy.html
  - [x] /engage/fr/vmware-to-openstack.html

Fixes #5674
